src/game/default/g_cmd.c

```diff
@@ -662,6 +662,46 @@ static const char *vote_cmds[] = {
 		"unmute",
 		NULL };
 
+static cvar_t *vote_cvars[32] = {}; // temp number so it doesn't have to be dynamically allocated
+
+/**
+ * @brief Create g_voting_allow_<vote> CVars
+ */
+void G_CreateVoteCVars() {
+	size_t i = 0;
+
+	while (vote_cmds[i]) {
+		const char *name = va("g_voting_allow_%s", vote_cmds[i]);
+		const char *desc = va("Allows voting on %s", vote_cmds[i]);
+
+		vote_cvars[i] = gi.Cvar(name, "1", CVAR_SERVER_INFO, desc);
+
+		i++;
+	}
+
+	vote_cvars[i] = NULL;
```

Comment:
Yep, would like to see this use the glib / GList API instead, but the helper is good style.